go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -47,6 +47,8 @@ var (
 	flagDumpHeadersCbor      = flag.String("dumpHeadersCbor", "", "Dump metadata and headers encoded as a canonical CBOR to a file for debugging.")
 	flagOutput               = flag.String("o", "out.sxg", "Signed exchange output file. If value is '-', sxg is written to stdout.")
 
+	flagNoCheck = flag.Bool("nocheck", false, "Do not reject invalid input arguments")
```

Comment:
I feel like `-nocheck=false` sounds confusing due to two negative words. Wouldn't `-skipcheck=true` be better?

Reply:
Or `flagCheck = flag.Bool("check", true, ...)`?

Reply:
Both are fine :-)

Reply:
Renamed to `-skipcheck`, as default-true boolean flag feels a bit unusual to me.